lib/Elevate/Components/SSH.pm

```diff
@@ -69,7 +69,7 @@ sub _check_ssh_config ($self) {
 
     if ( $setup !~ m{^\s*PermitRootLogin\b}m ) {
         WARN( <<~"EOS" );
-        OpenSSH configuration file does not explicitly state the option PermitRootLogin in sshd_config file, which will default in RHEL8 to "prohibit-password".
+        OpenSSH configuration file does not explicitly state the option PermitRootLogin in sshd_config file; this may default to "prohibit-password" after upgrading the distro.
```

Comment:
The OpenSSH configuration file does not explicitly state the PermitRootLogin option in the sshd_config file. This option may default to "prohibit-password" after the upgrade is complete.